src/docs/asciidoc/en/disruptor.adoc

```diff
@@ -216,7 +216,7 @@ It would be ideal if the graph of dependencies could be expressed without incurr
 While trying to address the problems described above, a design emerged through a rigorous separation of the concerns that we saw as being conflated in queues.
 This approach was combined with a focus on ensuring that any data should be owned by only one thread for write access, therefore eliminating write contention.
 That design became known as the "`Disruptor`".
-It was so named because it had elements of similarity for dealing with graphs of dependencies to the concept of "`Phasers`" footnote:phasers[Phasers - http://gee.cs.oswego.edu/dl/jsr166/dist/jsr166ydocs/jsr166y/Phaser.html] in Java 7, introduced to support Fork-Join.
+It was so named because it had elements of similarity for dealing with graphs of dependencies to the concept of "`Phasers`" footnote:phasers[Phasers - https://web.archive.org/web/20151117064926/http://gee.cs.oswego.edu/dl/jsr166/dist/jsr166ydocs/jsr166y/Phaser.html] in Java 7, introduced to support Fork-Join.
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Phaser.html Feels like the JDK7 Phaser doc link is more appropriate here?